.github/workflows/ci.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: [2.5, 2.6, 2.7]
+        ruby: [2.5, 2.6, 2.7, 3.0]
```

Comment:
It looks like GitHub is losing the `.0` off the end of this new value

<img width="190" alt="image" src="https://user-images.githubusercontent.com/497874/103149329-8e320a80-47bc-11eb-9a60-1af6a1fe62d9.png">

I think it's due to YAML interpreting these these values as floating point numbers. Best to quote them all as strings I think.

```suggestion
        ruby: ['2.5', '2.6', '2.7', '3.0']
```